mathinterpreter/__main__.py

```diff
@@ -1,6 +1,7 @@
-from mathinterpreter.interpreter import Interpreter
-from mathinterpreter.lexer import Lexer
-from mathinterpreter.parser import Parser
+#from mathinterpreter.interpreter import Interpreter
+#from mathinterpreter.lexer import Lexer
+#from mathinterpreter.parser import Parser
+from mathinterpreter.calculate import calculate
 
 
 def main():
```

Comment:
In my opinion there is no need anymore for a try block inside this main function, because the main problematic situations were move to function calculate(). Do you agree? If so, can you please remove this try-except block?